Makefile

```diff
@@ -24,6 +24,10 @@ TOOLS += trace-forwarder
 
 STANDARD_TARGETS = build check clean install static-checks-build test vendor
 
+# Variables for the build-and-publish-kata-debug target
+REGISTRY ?= ""
+TAG ?= ""
```

Comment:
Completely optional suggestion: Would it potentially help add clarity now and avoid issues later if there were named something like `KATA_DEBUG_REGISTRY` and `KATA_DEBUG_TAG`

Reply:
Makes sense, and I will add it.